fdbserver/worker.actor.cpp

```diff
@@ -613,6 +613,40 @@ ACTOR Future<Void> registrationClient(Reference<AsyncVar<Optional<ClusterControl
 	}
 }
 
+ACTOR Future<Void> peerHealthMonitor(Reference<AsyncVar<Optional<ClusterControllerFullInterface>>> ccInterface,
```

Comment:
Observability: How can SRE get the information: which worker thinks which workers are unavailable? 
In case the auto-detection does not work in prod, SRE can inspect the logic. 

(It does not have to be the v1 though. )